manifests/p/Project64/Project64/3.0.1/Project64.Project64.installer.yaml

```
@@ -0,0 +1,16 @@
+PackageIdentifier: Project64.Project64
+PackageVersion: 3.0.1
```

```suggestion
PackageVersion: 3.0.1.5664
```
This should match what is written to control panel. After updating this, you will need to rename the folder